### DISCUSSION TITLE
What is the password is not on the first line?

### DISCUSSION THREAD

**Discussion by mcepl** [languitar/pass-git-helper#433] [Q&A] [answered]
For fastmail I have this record in `pass(1)` (of course, all values are bogus):

```
verysecretpasswordforweb
login: mcepl@cepl.eu
recovery-code: something
thunderbird-password: something-else
davx5-password: yet-something-else
aerc-token: long-mysterious-string
aerc-addrs: aaaaaaaa
git-send-email-password: something-gitty
```

So, I would need pass-git-helper got the password via the equivalent of

```sh
 $ pass show mozilla/app.fastmail.com |awk '/git-send-email-password:/ {print $2}'
```

Is it possible?

**Comment by languitar** [comment, marked as answer]
There's a `specific_line` strategy that can extract certain lines, but only for the username. I also can't recommend that layout much. I'd used folders instead to organize this:

```
example.com/
  weblogin.gpg
  thunderbird-token.gpg
  ...
```

This makes all entries usable with most other tools, too.

**Reply by mcepl** [reply]
Perhaps you are right. Fastmail seem crazy in the same manner as Google: I am just in the second day of switching to them, and I have already this list of lines. Perhaps directory is a better way to organize. Thank you!
